### ISSUE TITLE
Burp Suite Free Edition fails to load

### ISSUE THREAD

**Issue by avi12** [agabrys/chocolatey-packages#28]
I'm on Windows 10 21H1, running PowerShell 5.1.19041.1023 via Windows Terminal.

After inspecting the install script, I saw the line
https://github.com/agabrys/chocolatey-packages/blob/882c328064719bc753a25c08d230183cef6973d1/manual/burp-suite-free-edition/noarch/tools/chocolateyInstall.ps1#L1
As it turns out, my PowerShell outputs an empty string.
```powershell
PS C:\Users\avi12> Split-Path -Parent $MyInvocation.MyCommand.Definition

PS C:\Users\avi12>
```

I recommend adding an additional check so that when it's empty, it attempts to execute `powershell.exe` as-is.

**Comment by agabrys** [comment]
How did you execute the script? Manually or by using `choco install burp-suite-free-edition`?

**Comment by avi12** [comment]
The latter, with the `-y` flag
The error I got is related to "JVM is not able to load" or something like that
I already installed Burp through the EXE

**Comment by agabrys** [comment]
```
 "JVM is not able to load" 
```

This means that JVM is not available on `PATH` or is broken. What happens when you execute `java -version`?

**Comment by avi12** [comment]
Not only JVM _is_ available, but also `$Env:JAVA_HOME` is

**Comment by agabrys** [comment]
The shortcut was created incorrectly. It expand the `JAVA_HOME` environment variable during its creation instead of using its value. I fixed it in 2021.8:

https://github.com/agabrys/chocolatey-packages/blob/882c328064719bc753a25c08d230183cef6973d1/manual/burp-suite-free-edition/noarch/tools/chocolateyInstall.ps1#L17

↓

https://github.com/agabrys/chocolatey-packages/blob/3cdc74f139ac22a501b40732cc0977b1db9e8bbe/manual/burp-suite-free-edition/noarch/tools/chocolateyInstall.ps1#L17

To fix it on your machine, you may open the shortcut file in Windows editor and update the path.
